managed/models/database_test.go

```diff
@@ -393,29 +393,34 @@ func TestDatabaseMigrations(t *testing.T) {
 			`INSERT INTO
 			nodes(node_id, node_type, node_name, distro, node_model, az, address, created_at, updated_at)
 			VALUES
-			('node_id', 'node_type', 'node_name', 'distro', 'node_model', 'az', 'address', '03/03/2014 02:03:04', '03/03/2014 02:03:04')`,
+			('node_id', 'generic', 'node_name', 'distro', 'node_model', 'az', 'address', '03/03/2014 02:03:04', '03/03/2014 02:03:04')`,
 		)
 		require.NoError(t, err)
 
 		// Insert dummy agent in DB
-		_, err = sqlDB.ExecContext(context.Background(),
-			`INSERT INTO
-			agents(mongo_db_tls_options, agent_id, agent_type, runs_on_node_id, created_at, updated_at, disabled, status, tls, tls_skip_verify, query_examples_disabled, max_query_log_size, table_count_tablestats_group_limit, rds_basic_metrics_disabled, rds_enhanced_metrics_disabled, push_metrics)
-			VALUES
-			('{"stats_collections": "db.col1,db.col2,db.col3"}', 'id', 'pmm-agent', 'node_id' , '03/03/2014 02:03:04', '03/03/2014 02:03:04', false, 'alive', false, false, false, 0, 0, false, false, false)`,
-		)
+		pmmAgent, err := models.CreatePMMAgent(db.Querier, "node_id", make(map[string]string))
+		require.NoError(t, err)
+
+		createdAgent, err := models.CreateAgent(db.Querier, models.NodeExporterType,
+			&models.CreateAgentParams{
+				PMMAgentID: pmmAgent.AgentID,
+				NodeID:     "node_id",
+				MongoDBOptions: &models.MongoDBOptions{StatsCollections: []string{
+					"db.col1", "db.col2", "db.col3",
+				}},
+			})
 		require.NoError(t, err)
 
 		// Apply migration
 		testdb.SetupDB(t, sqlDB, models.SkipFixtures, pointer.ToInt(63))
 
-		agent, err := models.FindAgentByID(db.Querier, "id")
+		agent, err := models.FindAgentByID(db.Querier, createdAgent.AgentID)
 		require.NoError(t, err)
-		require.Equal(t, "id", agent.AgentID)
+		require.Equal(t, createdAgent.AgentID, agent.AgentID)
 		require.Equal(t, []string{"db.col1", "db.col2", "db.col3"}, agent.MongoDBOptions.StatsCollections)
 	})
 	t.Run("stats_collections field migration: string array to string array", func(t *testing.T) {
-		sqlDB := testdb.Open(t, models.SkipFixtures, pointer.ToInt(63))
```

Comment:
it supposed to be 58 if I remember correctly

Reply:
Thanks. Test should be fixed now.

Reply:
it's supposed to be 58, not 68 :)

Reply:
@BupycHuk There is the problem with this test, it uses some old migrations, but our models are always represent last migration. So we should rewrite inserts here from reform to plain SQL, or drop this test at all. To be honest I don't see any value in testing one year old migration over and over again.